recordings-downloader/csharp/Recordings/Recordings.csproj

```diff
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{BB47FB92-3017-4E48-BA34-FF44D0F54348}</ProjectGuid>
+    <OutputType>Exe</OutputType>
+    <RootNamespace>Recordings</RootNamespace>
+    <AssemblyName>Recordings</AssemblyName>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
This is totally optional, but can you make this a .NET Core project? I was just playing around with core yesterday and was able to use VS Code with .NET Core to build and run a C# app on my mac. Would be neat if this can use core so it works for mac and windows users.